config/config.go

```diff
@@ -179,63 +120,3 @@ func (l *LocalStore) Fetch() ([]string, error) {
 	}
 	return l.RawKeys, nil
 }
-
-// ReadWriteRemoteRunnerConfig looks for an already existing remote config to read from, or asks the user to build one
-func ReadWriteRemoteRunnerConfig() (*RemoteRunnerConfig, error) {
```

Comment:
Should we write an example one for usage and place it with `framework.yaml` and `networks.yaml`?

Reply:
Added another readme, you can use both, YAML and env vars